Makefile

```diff
@@ -97,11 +99,32 @@ control-plane-fips-dev-docker: ## Build consul-k8s-control-plane FIPS dev Docker
 
 .PHONY: control-plane-test
 control-plane-test: ## Run go test for the control plane.
-	cd control-plane; go test ./...
+ifeq ("$(GOTESTSUM_PATH)","")
```

Comment:
I cribbed this conditional gotestsum running approach from the consul main repo to help when I was running and re-running the normal tests.